doc-checking-app-api/src/main/java/uk/gov/di/authentication/app/lambda/DocAppCallbackHandler.java

```diff
@@ -315,15 +316,20 @@ public APIGatewayProxyResponseEvent docAppCallbackRequestHandler(
                                 null,
                                 authenticationRequest.getResponseMode());
 
+                var metadataPairs = new ArrayList<AuditService.MetadataPair>();
+                metadataPairs.add(pair("internalSubjectId", AuditService.UNKNOWN));
+                metadataPairs.add(pair("isNewAccount", session.isNewAccount()));
+                metadataPairs.add(pair("rpPairwiseId", AuditService.UNKNOWN));
+                metadataPairs.add(pair("authCode", authCode));
+                if (authenticationRequest.getNonce() != null) {
+                    metadataPairs.add(pair("nonce", authenticationRequest.getNonce()));
```

Comment:
I've noticed this doesn't currently work. It's missing a `.getValue()` on the nonce